caplan/nytimes-android-martian-newsreader-OsayukiEhigie
# MARS-123: Add favorites

## What It Does
- Adds bookmark icon to Toolbar and each Article Preview Item
- Shows favorited articles and adds navigation
- Persists favorites in SharedPrefs

## How to Test
1. Open the Articles List
2. Select the bookmark icon to open the new Favorites screen, it should be empty at first
3. Return to Articles List and select an unfilled bookmark icon on an article
4. Observe that the selected bookmark icon is now filled
5. Open Favorites again
6. Observe that the saved article appears in this list
7. Select the filled bookmark icon
8. Return to Articles List
9. Observe that the bookmark icon on the previously favorited article is no longer filled
